__tests__/app.test.js

```diff
@@ -12,4 +20,24 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+
+  test("200: Responds with an array of topic objects with 'slug' and 'description' properties", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics).toBeInstanceOf(Array);
+        expect(topics).toHaveLength(3);
```

Comment:
This test is really important! Good to see 🥇 I like checking the instance as well above it 👍